great-idea-website/css/index.css

```diff
@@ -63,4 +63,112 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Your code starts here! */
\ No newline at end of file
+/* Your code starts here! */
+
+@import url('https://fonts.googleapis.com/css?family=Open+Sans+Condensed:300|Viga&display=swap');
+
+/* HEADER */
+.wrapper {
+	max-width: 876px;
+	margin: 0 20px;
```

Comment:
Like we talked about before on the 1:1 margin: 0 auto other than that looks perf!